config/environments/development.js

```diff
@@ -21,7 +21,9 @@ exports.getConfig = function() {
     database: 2,
 
     secret: 'secret',
-    origin: 'http://localhost:3333',
+
+    origin: '*',
```

Comment:
that's my stuff for testing, sorry

On Thu, Jul 30, 2015 at 5:58 PM, epicmonkey <notifications@github.com>
wrote:

> In config/environments/development.js
> <https://github.com/pepyatka/pepyatka-server/pull/277#discussion_r35879647>
> :
>
> > @@ -21,7 +21,9 @@ exports.getConfig = function() {
> >      database: 2,
> >
> >      secret: 'secret',
> > -    origin: 'http://localhost:3333',
> > +
> > +    origin: '*',
>
> Please, no.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pepyatka/pepyatka-server/pull/277/files#r35879647>.
>